llvm/include/llvm/Support/KnownBits.h

```diff
@@ -329,6 +329,19 @@ struct KnownBits {
   static KnownBits computeForSubBorrow(const KnownBits &LHS, KnownBits RHS,
                                        const KnownBits &Borrow);
 
+  /// Compute knownbits resulting from addition of LHS and RHS.
+  /// NSW and NUW flags are assumed to be false.
+  static KnownBits add(const KnownBits& LHS, const KnownBits& RHS) {
```

Comment:
Why not call it `operator+`?

Reply:
Mainly because I loathe operators for this kind of stuff :)

Reply:
Well we already use them for `&`, `|` and `^`...

Reply:
\*cough\* and we shouldn't \*cough\*